packages/uicore/src/components/Accordion/Accordion.tsx

```diff
@@ -20,6 +20,7 @@ export interface AccordionPanelProps {
   addDomId?: boolean
   disabled?: boolean
   className?: string
+  shouldRender?: () => boolean
```

Comment:
this should also support psasing a boolean directly.
`shouldRender?: boolean | () => boolean`